src/theory/quantifiers_engine.h

```diff
@@ -54,6 +54,8 @@ namespace quantifiers {
   class TermDb;
   class TermDbSygus;
   class TermUtil;
+  class Skolemize;
+  class TermEnumeration;
```

Comment:
Include corresponding header files.

Reply:
Same reason as mentioned in #1306, these headers are intentionally omitted.